controllers/getComments.controller.js

```diff
@@ -0,0 +1,19 @@
+const getAllComments = require('../models/getComments.model')
+const articleByID = require('../models/getArticleByID.model')
+
+function getComments (req, res, next) {
+    const { article_id } = req.params
+    articleByID(article_id).then((article) => {
+        getAllComments(article_id).then((comments) => {
+            res.status(200).send({ comments })
+        })
+        .catch((err) => {
+            next(err)
+        })
+    })
+    .catch((err) => {
+        next(err)
+    })
```

Comment:
The functionality of what you're doing here is great but don't nest promises. 
In your `.then` block you should return the next promise and chain on a `.then`. 

This keeps them sequential and allows you to have a single `.catch` at the end of your promise chain 👍